### ISSUE TITLE
"Recognised" HTML Tags? - Queries

### ISSUE THREAD

**Issue by pmario** [wikilabs/plugins#76]
### "Recognised" HTML Tags? - Queries

**1** - I noticed that one can directly in Custom Markup redefine "inline" elements as "block" elements and add classes ...

```
Start sentence, /°pre.c with this text wrapped in "pre"°/, then end the sentence.
```
... renders as ...
```
<p>Start sentence, <pre class="c wltc-inline wltc">
with this text wrapped in &quot;pre&quot;</pre>, then end the sentence.</p>
```
The advantage of the method **1** above is that _for occasional use you do NOT need to write pragma._

**2** - BUT also notice I CAN'T directly pass an "inline" element to replace the span. For instance this does not seem to work...
```
Start sentence, /°mark gives text still wrapped in the "span" element°/, then end the sentence.
```
... renders with `<span>` as ...
```
<p>Start sentence, <span class="wltc-inline wltc">gives text still wrapped in the &quot;span&quot; element</span>, then end the sentence.</p>
```

QUESTIONS...

1. _Is the behaviour of "2", above because there isn't a "lookup table for" "inline" elements in TW?_
2. I think it would be useful for me to make a reference table of which HTML tags are supported via the `<ID>[tag][.class] ` method.  _Should I use the list you pointed to before at_: [HTML Block Elements](https://github.com/Jermolene/TiddlyWiki5/blob/9716c326952c16f63345a135e73cf36670dca0d8/core/modules/config.js#L37) to help do that?

_Originally posted by @TiddlyTweeter in https://github.com/wikilabs/plugins/discussions/62#discussioncomment-356116_

**Comment by pmario** [comment]
will be fixed in V0.11.2

**Comment by TiddlyTweeter** [comment]
> will be fixed in V0.11.2

That is VERY good news! It would expand the utility a lot IMO.

**Comment by pmario** [comment]
V0.11.2 is uploaded already

**Comment by TiddlyTweeter** [comment]
> V0.11.2 is uploaded already

I been testing the new expansion to include "inline" tags. Seems to work well! Its neat.

I noticed that overall behaviour changed a bit. Previously "inline" Glyphs would directly accept "block" tags ... e.g. ....

> `/°div Created a div°/`

That behaviour seems to have _changed_ so that "inline Glyphs" only directly support "inline HTML tags" and no longer "HTML block tags". _Right?_ (I mean directly, pragma can still do it when needed.)

This not a criticism. But, in some ways, I did think, it might be better to **_let any glyph accept any HTML tag directly?_**

But, _IF we going to have the "inline Glyphs" only directly support inline HTML and "block Glyphs" only directly support block HTML, then I think it important the documentation gives a list of what tags work with which Glyphs??_
